boards/shields/dongle_screen/src/widgets/battery_status.c

```diff
@@ -92,11 +92,11 @@ static void set_battery_symbol(lv_obj_t *widget, struct battery_state state) {
         lv_label_set_text(label, "X");
     }
 
-    if (state.level < 1)
+    if (state.level <= CONFIG_DONGLE_SCREEN_BATTERY_RED_LEVEL)
     {
         lv_obj_set_style_text_color(label, lv_palette_main(LV_PALETTE_RED), 0);
-        lv_label_set_text(label, "X");
```

Comment:
@janpfischer i also changed this one, to actually display level when the red threshold reached. Let me know if you want me to get it back